### ISSUE TITLE
sb_get_event_filter() not working.

### ISSUE THREAD

**Issue by migvidal5** [Teamworksapp/smartabaseR#8]
sb_get_event_filter() does not seem to be filtering the data pulled from Smartabase. Instead it still seems to default and pull all data. 

I would expect either an error or an empty dataframe from the below code, instead it pulls all data over last 4 weeks.
---

Brief description of the problem

```r
df <- sb_get_event(
    form = "StatSports Manual Import",
    date_range = sb_date_range("4", "weeks"),
    url = "ussoccer.smartabase.com/ams",
    username = "username",
    password = "password",
    filter = sb_get_event_filter(user_key = "username", user_value = "john.smith")
  )
```

**Comment by zacpross** [comment]
Thank you, I can confirm this on my machine as well. I believe this a bug introduced via some of the caching work in v0.0.7. I'll have a patch up soon and let you know.

**Comment by zacpross** [comment]
@migvidal5 I just patched to 0.0.8, could you please re-install and let me know if you're still having issues?

**Comment by migvidal5** [comment]
@zacpross update looks to have resolved the issue. Thanks!
